### PR TITLE
Update ome-faq.yml

### PR DESCRIPTION
**Pull request by msbemba**
Added clarification that only encrypted messages sent internally to a shared mailbox will be viewable on the outlook client.  The delegates receiving it from external sources should open it via outlook on the web

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8455

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/ome-faq.yml [hunk 1]
```diff
@@ -216,7 +216,7 @@ sections:
       - question: |
           Can I open encrypted messages sent to a shared mailbox?
         answer: |
-          Yes! You can open encrypted messages for a shared mailbox when you're logged into a supported Outlook client.
+          Yes! You can open encrypted messages for a shared mailbox which sent internally within the same tenant, when you're logged into a supported Outlook client. Delegates to a shared mailbox getting  encrypted messages sent from an external organization will need to use Outlook on the web to sign in as the shared mailbox to read these messages.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ome-faq.yml]:
```suggestion
          Yes! You can open encrypted messages for a shared mailbox which sent internally within the same tenant when you're logged into a supported Outlook client. Delegates to a shared mailbox getting encrypted messages sent from an external organization should use Outlook on the web to sign in as the shared mailbox to read these messages.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@KCCross Gentle reminder. Thanks

File: microsoft-365/compliance/ome-faq.yml [hunk 2]
```diff
@@ -216,7 +216,7 @@ sections:
       - question: |
           Can I open encrypted messages sent to a shared mailbox?
         answer: |
-          Yes! You can open encrypted messages for a shared mailbox when you're logged into a supported Outlook client.
+          Yes! You can open encrypted messages for a shared mailbox which sent internally within the same tenant when you're logged into a supported Outlook client. Delegates to a shared mailbox getting encrypted messages sent from an external organization should use Outlook on the web to sign in as the shared mailbox to read these messages.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ome-faq.yml:219]:
```suggestion
          Yes! You can open encrypted messages for a shared mailbox which sent internally within the same tenant when you're logged into a supported Outlook client. Delegates to a shared mailbox getting encrypted messages sent from an external organization should use Outlook on the web to sign in as the shared mailbox to read these messages.
```

**Comment by KCCross** [comment]
Completed this work on the internal repo.

**Pull request closed without merging**